scripts/unix-install.sh

```diff
@@ -405,23 +408,26 @@ install_package()
   mkdir -p "$agent_home"
   succeeded
 
-  info "Creating plugins directory..."
-  mkdir -p "$agent_home/plugins"
-  succeeded
-
   info "Checking that service is not running..."
   stop_service
   succeeded
 
   info "Downloading binary..."
-  curl -L "$download_url" -o "$agent_binary" --progress-bar --fail || error_exit "$LINENO" "Failed to download package"
+  curl -L "$agent_download_url" -o "$agent_binary" --progress-bar --fail || error_exit "$LINENO" "Failed to download package"
   succeeded
 
   info "Setting permissions..."
   chmod +x "$agent_binary"
   ln -sf "$agent_binary" "/usr/local/bin/$BINARY_NAME"
   succeeded
 
+  info "Downloading plugins..."
+  curl -L "$plugins_download_url" -o plugins.tar.gz --progress-bar --fail || error_exit "$LINENO" "Failed to download plugins"
```

Comment:
We should probably download this to a temp directory, and remove it when we're done with it

Reply:
You could probably also just pipe directly to stdin of `tar`